.github/workflows/reassign-reviewer.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: actions-ecosystem/action-regex-match@d50fd2e7a37d0e617aea3d7ada663bd56862b9cc # v2.0.2
         with:
           text: ${{ github.event.comment.body }}
-          regex: '.*@modular-magician reassign[- ]reviewer ?@?([a-zA-Z0-9-_]+)?.*'
+          regex: '.*@modular-magician reassign[- ]+review[^@]*@?([a-zA-Z0-9-_]+)?.*'
```

Comment:
I think [^@] might be a little two permissive, we wouldn't want to include new lines in this capture.

Reply:
I've made it exclude newlines and carriage returns. It's possible to make it allow omitting the @ but I think requiring @ is preferable anyway.